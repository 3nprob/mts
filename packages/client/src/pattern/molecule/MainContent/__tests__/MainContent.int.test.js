import { defaultRender } from './MainContent.int.render'

describe('<MainContent />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
