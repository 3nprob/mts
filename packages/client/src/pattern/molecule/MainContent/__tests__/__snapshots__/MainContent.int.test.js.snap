// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContent /> Snaphot must match defaultRender 1`] = `
<main
  className="main"
>
  <div
    className="mainModule"
  >
    <div
      className="mainBackground"
    >
      <picture>
        <img
          alt="cover"
          className="mainBackgroundImage"
          src={Object {}}
        />
      </picture>
      <div
        className="mainBackgroundDimmer"
      />
      <div
        className="mainBackgroundCaption"
      >
        <h1
          className="mainBackgroundCaptionPrimaryHeader"
        >
          MTS
        </h1>
        <h2
          className="mainBackgroundCaptionSecondaryHeader"
        >
          React.js, Node.js and GraphQL
        </h2>
      </div>
    </div>
  </div>
</main>
`;
